File: java/PublicAPIClient/src/com/sdl/GraphQLErrorLocation.java
package com.sdl;

public class GraphQLErrorLocation {
    private int Line;
    private int Column;

    public int getLine() {
        return Line;
    }

    public void setLine(int line) {
        Line = line;
    }

    public int getColumn() {
        return Column;
    }

    public void setColumn(int column) {
        Column = column;
    }
}

File: net/src/Sdl.Tridion.Api.Client/GraphQLClient/Response/GraphQLErrorLocation.cs
namespace Sdl.Tridion.Api.GraphQL.Client.Response
{
    /// <summary>
    /// GraphQL Error locations.
    /// </summary>
    public class GraphQLErrorLocation
    {
        /// <summary>
        /// Error line number.
        /// </summary>
        public int Line { get; set; }

        /// <summary>
        /// Error column number.
        /// </summary>
        public int Column { get; set; }
    }
}
